packages/frontend/src/components/swap/components/TransactionDetails.js

```diff
@@ -171,34 +173,34 @@ const TransactionDetails = ({
         (tokenFrom.onChainFTMetadata.name === 'USN' || tokenTo.onChainFTMetadata.name === 'USN') 
         && CREATE_USN_CONTRACT;
 
-    function transactionDetailsSwitch(token) {
-        switch (token) {
-            case 'USN':
-                return (
-                    <TransactionDetailsUSN
-                        selectedTokenFrom={tokenFrom}
-                        selectedTokenTo={tokenTo}
-                        amount={amountTokenFrom}
-                        exchangeRate={exchangeRate}
-                        tradingFee={tradingFee}
-                        setSlippage={setSlippage}
-                    />
-                );
-            default:
-                return (
-                    <TransactionDetailsWrappedNear
-                        selectedTokenFrom={tokenFrom}
-                        selectedTokenTo={tokenTo}
-                        estimatedFeesInNear={`${
-                            amountTokenFrom > 1
-                                ? Math.trunc(amountTokenFrom).toString()
-                                : '1'
-                        }`}
-                        estimatedMinReceived={estimatedMinReceived}
-                    />
-                );
+    const getTransactionDetails = (tokenName) => {
+        if (showSlippageOption || tokenName === 'USN') {
+            return (
+                <TransactionDetailsUSN
```

Comment:
https://github.com/mynearwallet/near-wallet/pull/21#discussion_r979718133

Reply:
In the followed code I've changed this part. But in general we need to separate some code, because NEAR <> wNEAR is a different swap type than wNEAR <> NEP141. So we don't have **price impact, slippage, minimum received** in NEAR <> wNEAR